stubtest_allowlist.txt

```diff
@@ -0,0 +1,35 @@
+cheroot.ssl.pyopenssl.SSLConnection.accept
```

Comment:
What exactly does this file do?

Reply:
This class has generated members due the metaclass. I couldn't find proper type hints for these methods, therefore this file ignores the mentioned methods for now